scintillometry/fourier/tests/test_fourier.py

```diff
@@ -53,8 +56,8 @@ def test_fft(self, key):
         assert np.allclose(ifft(Y), self.y_exp, **self.tolerances)
 
         # Check frequency.
-        assert_quantity_allclose(fft.freq, np.fft.fftfreq(
-            len(self.y_exp), d=(1. / self.sample_rate)))
+        assert_quantity_allclose(fft.freq.squeeze(), self.freq_Y_exp)
```

Comment:
No need for the squeeze is this is the expected shape.

Reply:
Isn't `self.freq_Y_exp` a different shape though?  Or should I expand those in the setup instead of also checking the shape below line 59?

Reply:
Never mind, just saw below.